public/app/core/components/Footer/Footer.tsx

```diff
@@ -16,22 +16,22 @@ export const Footer: FC<Props> = React.memo(
         <div className="text-center">
           <ul>
             <li>
-              <a href="http://docs.grafana.org" target="_blank">
+              <a href="http://docs.grafana.org" target="_blank" rel="noopener">
```

Comment:
Out of curiosity, how this relates to performance improvements?

Reply:
https://jakearchibald.com/2016/performance-benefits-of-rel-noopener/